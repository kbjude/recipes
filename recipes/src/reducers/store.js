import { applyMiddleware, createStore } from 'redux';
import thunk from 'redux-thunk';
import rootReducer from '.';

const middlewares = [thunk];

const store = createStore(rootReducer, applyMiddleware(...middlewares));
export default store;
