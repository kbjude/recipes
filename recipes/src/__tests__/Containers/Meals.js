import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../../reducers/store';
import Meals from '../../containers/Meals';

describe('Display', () => {
  it('renders correctly', () => {
    const snap = renderer.create(
      <BrowserRouter>
        <Provider store={store}>
          <Meals
            match={{ params: { name: 'Beef' } }}
          />
        </Provider>
      </BrowserRouter>,
    ).toJSON();
    expect(snap).toMatchSnapshot();
  });
});
