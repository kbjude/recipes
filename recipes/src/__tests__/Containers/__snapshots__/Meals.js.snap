// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display renders correctly 1`] = `null`;
